Makefile

```diff
@@ -399,7 +399,7 @@ staticcheck:
 test-proto-static:
 	./etc/proto/test_no_changes.sh || echo "Protos need to be recompiled; run make proto-no-cache."
 
-test-deploy-manifests:
+test-deploy-manifests: install
```

Comment:
I feel like the install should also apply when doing a regenerate. You could just add another make target for regenerating.

Reply:
:+1: Added a new make target